openenergysolutions/dss_rs
# Ocp 2398

Will add unit-tests for the following scenarios:
- open/close breaker
- open/close capbank
- enable/disable/set p/q/pf load
- enable/disable set p/q/pf generator
- tap reg up/tap reg down
- open/close recloser
- open/close switch

As well as add CI/CD to verify builds/tests.
